packages/web/src/components/user-badges/TieredUserBadge.tsx

```diff
@@ -55,13 +68,23 @@ export const TieredUserBadge = (props: TieredUserBadgeProps) => {
   }
 
   return (
-    <AudioHoverCard
-      tier={tier}
-      amount={formattedBalance}
-      anchorOrigin={anchorOrigin}
-      transformOrigin={transformOrigin}
+    <Box
+      onClick={(e: MouseEvent) => e.stopPropagation()}
```

Comment:
nit: maybe worth useCallback-ing this if it's a component that's gonna be rendered in a lot of places?